src/test/java/com/celements/cleverreach/exception/CssInlineExceptionTest.java

```diff
@@ -0,0 +1,803 @@
+      + "        at ch.programmonline.proz.TagesagendaCleverReachUpdateJob.executeJob(TagesagendaCleverReachUpdateJob.java:69)\n"
+      + "        at com.celements.scheduler.job.AbstractJob.execute(AbstractJob.java:80)\n"
+      + "        at org.quartz.core.JobRunShell.run(JobRunShell.java:202)\n"
+      + "        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:525)\n"
+      + "Nested exception:\n"
+      + "org.xml.sax.SAXParseException; lineNumber: 247; columnNumber: 114; The entity name must immediately follow the '&' in the entity reference.\n"
+      + "        at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source)\n"
+      + "        at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source)\n"
+      + "        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)\n"
+      + "        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)\n"
+      + "        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)\n"
+      + "        at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)\n"
+      + "        at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)\n"
+      + "        at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)\n"
+      + "        at org.dom4j.io.SAXReader.read(SAXReader.java:465)\n"
+      + "        at com.celements.cleverreach.util.DefaultCssInliner.prepareInput(DefaultCssInliner.java:65)\n"
+      + "        at com.celements.cleverreach.util.DefaultCssInliner.inline(DefaultCssInliner.java:48)\n"
+      + "        at com.celements.cleverreach.util.DefaultCssInliner.inline(DefaultCssInliner.java:39)\n"
+      + "        at com.celements.cleverreach.MailingConfig.getContentHtmlCssInlined(MailingConfig.java:88)\n"
+      + "        at com.celements.cleverreach.CleverReachRest.updateMailing(CleverReachRest.java:82)\n"
+      + "        at ch.programmonline.proz.TagesagendaCleverReachUpdateJob.executeJob(TagesagendaCleverReachUpdateJob.java:69)\n"
+      + "        at com.celements.scheduler.job.AbstractJob.execute(AbstractJob.java:80)\n"
+      + "        at org.quartz.core.JobRunShell.run(JobRunShell.java:202)\n"
+      + "        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:525)";
+
+  private static final String EXCEPTION_CAUSE_5_LINES1 = "        <td><div class=\"td\">\n"
```

Comment:
can you please move the very long text to test/resources/ as text-files and loading if neaded?